README.md

```diff
@@ -145,6 +145,14 @@ For each field (question) that needs to be indexed, there must be a `filter_` pr
 same as the filename, then the _overridden_ `id` should be used here. (Compare with the manifest,
 where in all cases the filename should be used to specify the question.)
 
+We are using Elasticsearch multifields to support service aggregations by specific attributes (eg lot and category).
+We use a `raw` subfield so that tokenization is not performed on the values to make aggregation possible with natural
```

Comment:
Yes we should do that. My bad. Internally I just find multifield such a weird term.